eta/core/image.py

```diff
@@ -485,6 +486,32 @@ def is_valid_object(self, obj):
         except (ImageLabelsSchemaError, AttributeContainerSchemaError):
             return False
 
+    def is_valid_subset_schema(self, schema):
+        '''Validates that the provided schema is a subset of self
+        
+        Args:
+            schema: an ImageLabelsSchema
+        Returns:
+            True/False
+        '''
+
+        # Validate frame attributes
+        for attr in schema.attrs:
+            if not self.is_valid_image_attribute(attr):
```

Comment:
This doesn't work. `attr` is an `AttributeSchema`, but `is_valid_image_attribute` expects an `Attribute`

Reply:
The same comment applies throughout this PR. No functionality yet exists for, e.g., checking whether the `categories` of one `CategoricalAttributeSchema` are a subset of another. That will have to be written

Reply:
Ah, gotcha. Thanks for the review. I'll work it out.